offchain-modules/packages/x/src/ckb/tx-helper/multisig/deploy.ts

```diff
@@ -54,16 +54,16 @@ function getMultiSigCellCapacity() {
   return min;
 }
 
-async function deploy() {
-  const fromPrivateKey = parsePrivateKey(ForceBridgeCore.config.ckb.fromPrivateKey);
-  const fromAddress = getFromAddr();
-  const multisigLockScript = getMultisigLock(ForceBridgeCore.config.ckb.multisigScript);
-  console.log(`multisigLockScript: ${JSON.stringify(multisigLockScript, null, 2)}`);
-  const multisigAddress = getMultisigAddr(ForceBridgeCore.config.ckb.multisigScript);
+async function deploy(ckbPrivateKey: string, multisigItem: MultisigItem) {
```

Comment:
async function deploy(...): Promise<CkbConfig>

Reply:
tried to add type annotation and found that we mixed usage of pw-core Script and lumos Script. There are too many files to edit, leave it to next PR.